recipes/redbiom/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "redbiom" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/biocore/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: b5a1f8a8e6d81159fbe639cffd2aff0abffc769247997ff19f1f36755c1f9da7
+
+build:
+  skip: true  # [win or py2k]
```

Comment:
Could this package be `noarch`? Doesn't seem to have compiled code.

Reply:
`biom-format` and `scikit-bio` both do and don't support windows. I did relax this constraint though as I don't *think* it needs to be explicitly enforced here.